standard-app/example.values.yaml

```diff
@@ -1,6 +1,7 @@
 image: us-central1-docker.pkg.dev/cloudkite-infra-ops/cloudkite-docker-images/example-image
 tag: example_tag
 
+enableConfigMap: true
```

Comment:
It makes no sense to use `enableConfigMap` key when you have consecutive conditions checks afterwards
```
{{- if .Values.enableConfigMap }}
{{- if $appConfig.volumes }}
{{- range $appConfig.volumes }}
{{- if .type.configMap }}
```